installer/sharch_body.sh

```diff
@@ -25,13 +25,19 @@ fi
 
 echo " OK."
 
+image_size=$(( $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) / 1024))
```

Comment:
>/ [](http://example.com/codeflow?start=84&length=1)

Do you want to get the ceiling of division?

Reply:
okay.will update with ceiling.

Reply:
Sorry, how do you get the ceiling? Also there are 2 more division below.

Instead of `x / 1024`,
`(x + 1023) / 1024` will be much safer.

Reply:
we can avoid ceiling here as we already adding  ceiling to image_size  by 1G  in this command  " mount_size=$(((image_size/1024/1024)+1)) " .Here if we add it will be ceiled by 1KB only ,so it is not necessary.

Reply:
Okay, you are using padding instead of ceiling. Do you think 1G padding is too large?

Reply:
"mount_size=$(((image_size/1024/1024)+1))" Here we are handling ceiling for image_size  and extra free space for the mount , so extra free space will be less than 1GB.

Reply:
My only concern is the extreme padding size 1G seem too large. Others look good to me.

Reply:
@qiluo-msft, 
is it fine to update 32MB extra space? Find below changes. If this is fine, we will push the changes.

mount_size=$((((image_size+1023)/1024) + 32))
mount -o remount,size="${mount_size}M" -t tmpfs tmpfs-installer $tmp_dir || exit 1

Reply:
32M padding is good. However you divided 3 times, so I think you formula will not work for corner cases.
This is my math
```
bytes = $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) 
image_size=$((bytes + 1023) / 1024)
mount_size=$((image_size + 31) / 32) * 32
mount -o remount,size="${mount_size}M"
```

Reply:
> 32M padding is good. However you divided 3 times, so I think you formula will not work for corner cases.
> This is my math
> 
> ```
> bytes = $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) 
> image_size=$((bytes + 1023) / 1024)
> mount_size=$((image_size + 31) / 32) * 32
> mount -o remount,size="${mount_size}M"
> ```

Updated PR with your comments. Please review.